Correction on the consistency of design and buttons #13958

Ibochkarev opened the pull request:
### What does it do?
Improves the consistency of the design, and the color of the buttons and their state when you hover the mouse

### Why is it needed?
Change the colors of the buttons

After fix:
![After](https://user-images.githubusercontent.com/2138260/51368572-32639f00-1b1a-11e9-9ec8-7c4f8073720a.gif)

### Related issue(s)/PR(s)
Issue #13958

Ibochkarev committed:
Correction on the consistency of design and buttons #13958

Ibochkarev committed:
Small fix

alroniks commented on _build/templates/default/sass/_colors-and-vars.scss:
```diff
@@ -88,19 +89,14 @@ $buttonColorPrimary: $white;
-$buttonPrimaryBg2Hover: darken($buttonPrimaryBg2, 6%);
-$buttonPrimaryBg1Active: darken($buttonPrimaryBg1, 12%);
-$buttonPrimaryBg2Active: darken($buttonPrimaryBg2, 12%);
+$buttonPrimaryBgActive: darken($green, 12%);
```
Make sense to use `$darkGreen` variable here.

Ibochkarev replied:
Thanks

alroniks commented on _build/templates/default/sass/_colors-and-vars.scss:
```diff
@@ -20,6 +20,7 @@ $yellow: #FFEA80;
 $orange: #F49548;
 $red: #BE0000;
 $green: #6CB24A;
+$darkGreen: darken($green, 12%);
```
```suggestion
$greener: darken($green, 12%);
```

alroniks reviewed (commented):
My questions are not strictly required, just some notices.

alroniks committed:
Update _build/templates/default/sass/_colors-and-vars.scss

Co-Authored-By: Ibochkarev <ivanx86@gmail.com>

Ibochkarev committed:
Change name var color

Ibochkarev committed:
Merge branch 'fix#13958' of https://github.com/Ibochkarev/revolution into fix#13958

Ibochkarev committed:
Small fix

Ibochkarev commented:
@JoshuaLuckers @Jako Please check this PR

JoshuaLuckers commented:
Thanks for notifying. I will check the PR when I have time, in the meantime would you mind fixing the conflict?

Ibochkarev committed:
Merge branch '3.x' into fix#13958

Ibochkarev commented:
@JoshuaLuckers Conflicts fix.

Ibochkarev committed:
Update scss

Update _navbar.scss

Update _build/templates/default/sass/_colors-and-vars.scss

Co-Authored-By: Ibochkarev <ivanx86@gmail.com>

Small fix

Ibochkarev committed:
Delete header.php

OptimusCrime commented:
Why was this closed?

Ibochkarev commented:
@OptimusCrime I was wrong in working with git. I will fix some points and today I will open a new pull request
